runtime/near-vm-logic/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "near-vm-logic"
-version = "0.8.0"
+version = "0.9.0"
```

Comment:
Please update version of all crates in `nearcore/runtime`. We currently keep them in sync, until we decide to split them in the future.

Reply:
I see